image_width: 640
image_height: 480
camera_name: camera
camera_matrix:
  rows: 3
  cols: 3
  data: [519.1462289999999, 0, 331.595942, 0, 518.756583, 238.369554, 0, 0, 1]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.008297000000000001, -0.040748, 0.001068, -0.001163, 0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1, 0, 0, 0, 1, 0, 0, 0, 1]
projection_matrix:
  rows: 3
  cols: 4
  data: [517.754395, 0, 331.098855, 0, 0, 518.769104, 238.733157, 0, 0, 0, 1, 0]
